cleansweep/templates/admin/sendmail.html

```diff
@@ -1,25 +1,52 @@
+{% extends "base.html" %}
 {% from "macros.html" import render_field %}
 
-
-{% extends "place.html" %}
-
-{% block subnav %}
-  {{ subnav(place, tab="sendmail") }}
+{% block title %}
+    {{ get_site_title() }} - Send E-mails
 {% endblock %}
 
-{% block page_content %}
-  <h2>Send E-mails</h2>
+{% block extrahead %}
+    <script src='//cdn.tinymce.com/4/tinymce.min.js'></script>
+    <script>
+        tinymce.init({
+            selector: '#email',
+            theme: 'modern',
+            plugins: [
+                'advlist autolink link image lists charmap print preview hr anchor pagebreak',
+                'searchreplace wordcount visualblocks visualchars code fullscreen insertdatetime media nonbreaking',
+                'save table contextmenu directionality emoticons template paste textcolor legacyoutput'
+            ],
+            content_css: 'css/content.css',
+            toolbar: 'insertfile undo redo | styleselect | bold italic | alignleft aligncenter alignright alignjustify | bullist numlist outdent indent | link image | print preview media fullpage | forecolor backcolor emoticons',
+            browser_spellcheck: true
+        });
+    </script>
+{% endblock %}
 
-  {% if sent %}
-    Done! The email with subject <strong>{{ form.subject.data }}</strong> will be sent to selected people shortly.
-  {% else %}
-    <form id="sendmail-form" role="form" method="POST">
-      {{ form.csrf_token }}
-      {{ render_field(form.people) }}
-      {{ render_field(form.subject) }}
-      {{ render_field(form.message) }}
-      <button type="submit" class="btn btn-default">Send</button>
-    </form>
-  {% endif %}
+{% block content_head %}
+    <div class="page-content">
+        <ul class="breadcrumb breadcrumb-collapse">
+            <li><a href="{{ url_for('place', place=place) }}">{{ place.name }}</a></li>
+            <li></li>
+        </ul>
+        <h2>Send E-mails</h2>
+    </div>
+{% endblock %}
 
+{% block content_body %}
+    {% if sent %}
+        Done! The email with subject <strong>{{ form.subject.data }}</strong> will be sent to selected people
+        shortly.
+    {% else %}
+        <form id="sendmail-form" role="form" method="POST">
+            {{ form.csrf_token }}
+            {{ render_field(form.people) }}
+            {{ render_field(form.subject) }}
+            <dl>
+                <dt>{{ form.message.label }}</dt>
+                <dd>{{ form.message(rows=15) }}</dd>
```

Comment:
You can pass id here. 

Also, why are you using `<dl>` here instead of using the `render_field`? You can call it as `render_field(form.message, rows=15, id="email")`. 

Could you change the id to email-message or something. Email sounds like an input for email address.